BUILDING-cmake.md

```diff
@@ -78,25 +93,35 @@ CMake has no built-in way of printing all available configuration options. You c
 top-level `CMakeLists.txt` which contains a block of `option` and `cmake_dependent_option` commands, or use one of the
 available CMake UIs which will display the options after configuring the project once.
 
-### Boolean switches
+### Important build switches
 
-The following table also gives you an overview of the available options and their defaults. All options accept a boolean
+The following table also gives you an overview of important build options and their defaults. All options accept a
+boolean
```

Comment:
Could mention how to change the switches (`ccmake .` is it?)

Reply:
There's also the Qt-based CMake GUI which can do it, yet I'd rather not go too deeply into CMake details in the build instructions as it's all documented on the CMake website. And depending on the OS and CMake version installed, none of those frontend may even be available. Deleting the build dir (or at least `CMakeCache.txt` inside it) and freshly reconfiguring the build is better in most cases, as fiddling around with the cache can break the build system.

Besides that, most people probably won't build libprojectM themselves, or with default settings. We can still put a link to the CMake website in a prominent place, in case someone can't find it on google.